android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -145,6 +145,7 @@ class XMTPModule : Module() {
             signer = reactSigner
             val options = ClientOptions(api = apiEnvironments(environment, appVersion))
             clients[address] = Client().create(account = reactSigner, options = options)
+            ContentJson.Companion
```

Comment:
I'm not sure it's necessary to call Companion here it could be implicitly called without it. But I wanted to explicit that this is triggering the `init` function that registers the codecs.